### ISSUE TITLE
Some problems with events: click, mouseup, mousedown, and change for Chrome 

### ISSUE THREAD

**Issue by fresheneesz** [bitovi/syn#76]
Chrome 40.0.2214.111 m

### `click` event

`event.originalEvent.fromElement should = null vs <html>
event.screenX and event.screenY don't seem to match the values clientX/Y, pageX/Y, etc 

### `mousedown` and `mouseup` events

`event.originalEvent.__proto__` should = <KeyBoardEvent> vs <Event>
`event.view` should = <Window> vs `undefined`

the following are missing entirely:
`event.originalEvent.detail`: 0
`event.originalEvent.keyIdentifier`: "U+xx58"
`event.originalEvent.keyLocation`: 0
`event.originalEvent.layerX`: 0
`event.originalEvent.layerY`: 0
`event.originalEvent.location`: 0
`event.originalEvent.pageX`: 0
`event.originalEvent.pageY`: 0
`event.originalEvent.repeat`: false
`event.originalEvent.view`: <Window>

### `change` event

I couldn't get the change event to fire at all. It doesn't fire if you type in an input field then click somewhere else:

 syn.type($("#b"), "x")
 syn.click($("#a"))

Even if one of those events is done manually, the change event doesn't happen. I also noticed that focus is not changed on click - it should be.

**Comment by justinbmeyer** [comment]
```
syn.type($("#b"), "x")
```

should be

```
syn.type($("#b")[0], "x")
```

**Comment by fresheneesz** [comment]
That's true.. but its funny, it still types stuff, just doesn't send the change event. When I correct it, it has exactly the same behavior - no change event.

**Comment by fresheneesz** [comment]
Are you able to repro?

**Comment by justinbmeyer** [comment]
I've not tried.  However, if it's breaking for you, I can show you how to create a test for it.
